### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by kpeeples**

### CODE REVIEW

File: containers/docker/Dockerfile [hunk 1]
```diff
@@ -10,92 +12,80 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
...
+# from postbuild
+# https://github.com/opencybersecurityalliance/kestrel-huntbook/blob/main/.binder/postBuild
+################################
+# 1. Setup Git LFS - to setup gitlab-lfs for this container it has to be grabbed from the site as git-lfs is not in the repo for the base
+################################
+# analytics prereq
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+RUN sudo apt-get install --yes git-lfs
 RUN mkdir -p /opt/kaas && chown ${NB_UID}:${NB_UID} /opt/kaas
+COPY kaas-logo-san-seriff.png /opt/kaas/kaas-logo-san-seriff.png
```
Comment (subbyte) [inline containers/docker/Dockerfile]: I get an error from here.

File: containers/docker/Dockerfile [hunk 2]
```diff
@@ -10,92 +12,80 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
...
-RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-
-# install git lfs
-RUN apt-get install --yes git-lfs
-
+# install prerequisites
+# 1. from requirements.txt for jupyter hub
+# https://github.com/opencybersecurityalliance/kestrel-huntbook/blob/main/.binder/requirements.txt
+ # analytics::dataexfiltration analytics::log4shell analytics::piniponmap analytics::sklearn-cluster analytics::attribute-plot analytics::suspiciousscoring 
+RUN apt-get install --yes flask flask-cors tabulate urllib3 geoip2 folium scikit-learn gower business-rules matplotlib
```
Comment (subbyte) [inline containers/docker/Dockerfile]: These are Python packages, need to install via `pip`.

**Review by subbyte — COMMENTED** [review]
I tried the Dockerfile to build a container but failed.

**Review by subbyte — CHANGES_REQUESTED** [review]
Happy New Year @kpeeples !

Great to see the tutorials are included in this version!

Two minor issues need to be addressed so all tutorial huntbooks (especially the analytics ones in huntbook 5) can work:

1. `apt-get install --yes dnsutils whois` (https://github.com/opencybersecurityalliance/kestrel-huntbook/blob/main/.binder/apt.txt#L4)
2. copy/link the `GeoLite2-City.mmdb` file from `data-bucket-kestrel` repo folder to `kestrel-analytics` repo folder (https://github.com/opencybersecurityalliance/kestrel-huntbook/blob/main/.binder/postBuild#L27C33-L27C51)

**Comment by kpeeples** [comment]
I added those two and posted the dockerfile changes but looks like it ended up as a new PR.  These are from the container testing.

File: containers/docker/Dockerfile [hunk 3]
```diff
@@ -1,101 +1,115 @@
...
-    ln -s /home/jovyan/.config/kestrel/stixshifter.yaml /opt/kaas/kestrel-huntbook/blackhat22/stixshifter.yaml
-
-RUN rmdir /opt/kaas/kestrel-huntbook/config
-
+RUN git clone https://github.com/opencybersecurityalliance/kestrel-analytics.git /opt/kaas/kestrel-analytics
+RUN ln -s /opt/kaas/data-bucket-kestrel /home/jovyan/data-bucket-kestrel 
+RUN ln -s /opt/kaas/kestrel-huntbook /home/jovyan/kestrel-huntbook 
+RUN ln -s /opt/kaas/kestrel-analytics /home/jovyan/kestrel-analytics 
+RUN cp /opt/kaas/data-bucket-kestrel/stix-bundles/lab101.json /tmp
+RUN cp /opt/kaas/data-bucket-kestrel/GeoLite2/GeoLite2-City.mmdb /opt/kaas/kestrel-analytics/GeoLite2-City.mmdb
```
Comment (subbyte) [inline containers/docker/Dockerfile]: Oops, the destination dir should be `/opt/kaas/kestrel-analytics/analytics/piniponmap`.

**Review by subbyte — CHANGES_REQUESTED** [review]

File: containers/docker/Dockerfile (hunk 3 above)
Comment (kpeeples) [reply containers/docker/Dockerfile]: modified

**Review by subbyte — APPROVED** [review]

**Pull request merged**